spring-cloud-function-adapters/spring-cloud-function-adapter-gcp/src/test/java/org/springframework/cloud/function/adapter/gcloud/integration/CloudFunctionServer.java

```diff
@@ -82,13 +76,6 @@ protected void before() throws InterruptedException {
 
 		this.serverThread = new Thread(startServer);
 		this.serverThread.start();
-
-		// Wait for the server to start up.
-
-		TestRestTemplate template = new TestRestTemplate();
-		await().atMost(5, TimeUnit.SECONDS)
```

Comment:
This didn't do anything for me. Is it really necessary?
@dzou

Reply:
I see, I added that because the test would flake for me when I ran it with `mvn test` on the command line. Running through intellij didn't cause problems for me though.

Reply:
Did this top the flakiness?

Reply:
Yeah it stopped the flakiness when I ran it from command line via `mvn test`